SPECS/zeromq/zeromq.spec

```diff
@@ -18,24 +18,27 @@ queues, multiple messaging patterns, message filtering (subscriptions), seamless
 access to multiple transport protocols and more.
 
 %package    devel
-Summary:    Header and development files for zeromq
-Requires:   %{name} = %{version}
+Summary:        Header and development files for zeromq
+Requires:       %{name} = %{version}
+
 %description    devel
 It contains the libraries and header files to create applications
 
 %prep
-%setup -q -n libzmq-%{version}
+%autosetup -n zeromq-%{version} -p1
```

Comment:
is the libzmq -> zeromq change ok?

Reply:
Yes; this is how libzmq seems to be publishing source tarballs right now. It's a little unclear when that change happened, but the previous releases also use this naming scheme.